src/azure-cli/azure/cli/command_modules/databoxedge/generated/_help.py

```diff
@@ -0,0 +1,332 @@
+    short-summary: Manage bandwidth schedule with databoxedge
+"""
+
+helps['databoxedge bandwidth-schedule list'] = """
+    type: command
+    short-summary: "Gets all the bandwidth schedules for a Data Box Edge/Data Box Gateway device."
+    examples:
+      - name: BandwidthScheduleGetAllInDevice
+        text: |-
+               az databoxedge bandwidth-schedule list --device-name "testedgedevice" --resource-group \
+"GroupForEdgeAutomation"
+"""
+
+helps['databoxedge bandwidth-schedule show'] = """
+    type: command
+    short-summary: "Gets the properties of the specified bandwidth schedule."
+    examples:
+      - name: BandwidthScheduleGet
+        text: |-
+               az databoxedge bandwidth-schedule show --name "bandwidth-1" --device-name "testedgedevice" \
+--resource-group "GroupForEdgeAutomation"
+"""
+
+helps['databoxedge bandwidth-schedule create'] = """
+    type: command
+    short-summary: "Create a bandwidth schedule."
+    examples:
+      - name: BandwidthSchedulePut
+        text: |-
+               az databoxedge bandwidth-schedule create --name "bandwidth-1" --device-name "testedgedevice" --days \
+"Sunday" "Monday" --rate-in-mbps 100 --start "0:0:0" --stop "13:59:0" --resource-group "GroupForEdgeAutomation"
+"""
+
+helps['databoxedge bandwidth-schedule update'] = """
+    type: command
+    short-summary: "Update a bandwidth schedule."
+"""
+
+helps['databoxedge bandwidth-schedule delete'] = """
+    type: command
+    short-summary: "Deletes the specified bandwidth schedule."
+    examples:
+      - name: BandwidthScheduleDelete
+        text: |-
+               az databoxedge bandwidth-schedule delete --name "bandwidth-1" --device-name "testedgedevice" \
+--resource-group "GroupForEdgeAutomation"
+"""
+
+helps['databoxedge bandwidth-schedule wait'] = """
+    type: command
+    short-summary: Place the CLI in a waiting state until a condition of the databoxedge bandwidth-schedule is met.
+    examples:
+      - name: Pause executing next line of CLI script until the databoxedge bandwidth-schedule is successfully \
+created.
+        text: |-
+               az databoxedge bandwidth-schedule wait --name "bandwidth-1" --device-name "testedgedevice" \
+--resource-group "GroupForEdgeAutomation" --created
+      - name: Pause executing next line of CLI script until the databoxedge bandwidth-schedule is successfully \
+updated.
+        text: |-
+               az databoxedge bandwidth-schedule wait --name "bandwidth-1" --device-name "testedgedevice" \
+--resource-group "GroupForEdgeAutomation" --updated
+      - name: Pause executing next line of CLI script until the databoxedge bandwidth-schedule is successfully \
+deleted.
+        text: |-
+               az databoxedge bandwidth-schedule wait --name "bandwidth-1" --device-name "testedgedevice" \
+--resource-group "GroupForEdgeAutomation" --deleted
+"""
+
+helps['databoxedge'] = """
+    type: group
+    short-summary: Manage job with databoxedge
+"""
```

Comment:
duplicate?

Reply:
I set group `databoxedge job` to group `databoxedge` using cli-directive as there contains only one command `show-job`. Then the `autorest --az` generate the help for `databoxedge job` with the name `databoxedge`. So there are several help info for `databoxedge`. 

Actually I have overwrited help for `databoxedge` under manual folder.